superset/cccs/pythonpath/jinja_context_addons.py

```diff
@@ -118,17 +153,33 @@ def render_ipv4_either_number_columns(src_col, dst_col, val):
     # ( SRC_IP = 22345 OR (SRC_IP >= 10000 AND SRC_IP <= 20000) )
     # OR
     # ( DST_IP = 22345 OR (DST_IP >= 10000 AND DST_IP <= 20000) )
-    return ' OR '.join(render_in_conditions(val, src_col)
-                     + render_in_conditions(val, dst_col))
-    
+    return " OR ".join(
+        render_in_conditions(val, src_col) + render_in_conditions(val, dst_col)
+    )
 
 
+def dashobard_link(link_label, dashboard_id, src_column, target_column):
+    from requests.utils import requote_uri
+
+    # return requote_uri('test' + str(dashboard_id) + 'asldfjlf')
+
+    prefix = (
+        '<a href="http://10.162.232.22:8088/superset/dashboard/'
```

Comment:
This looks like a sketchy hard-coded value. If this is a just-for-now thing, could we tag it with a FIXME comment or something?

Reply:
I put a FIXME